app/models/pawn.rb

```diff
@@ -1,5 +1,40 @@
 class Pawn < Piece
 
+  def valid_move?(x, y)
+    unless super(x, y)
+      return false
+    end
+    x_distance = x_position - x
+    y_distance = y_position - y
+    if occupied?(x, y)
+      if color == 'white'
+        x_distance.abs == 1 && y_distance == 1
+      else
+        (x_distance).abs == 1 && -y_distance == 1
+      end
+    elsif color == 'white'
+      if ! has_moved
```

Comment:
haha i guess what I'm saying that you don't need `unless` or `!` if you just switch what happens

i.e.
```
if true
  foo 
else 
  bar 
end
```
is easier to read than
```
if !true
  bar
else
  foo
end
```